src/routes/opening/index.tsx

```diff
@@ -285,15 +286,15 @@ export const SafeDeployment = ({
           </BodyInstruction>
         )}
 
-        {steps[stepIndex].instruction && creationTxHash ? (
```

Comment:
Why was the first flag removed?

Reply:
For future reference: every step has this.

Reply:
The last step has it set as undefined so I think this check was here to not display the transaction hash
![Screenshot 2022-04-11 at 18 08 54](https://user-images.githubusercontent.com/5880855/162783740-306506cc-c259-4c3b-9590-b08ebd02b10e.png)

Reply:
Good catch, I'll revert it then.